lib/Hooks.re

```diff
@@ -238,7 +238,7 @@ module Effect = {
           state.previousCondition = condition;
           Some(
             () => {
-              ignore(executeOptionalHandler(cleanupHandler));
+              executeOptionalHandler(cleanupHandler);
```

Comment:
Luckily this had just been made unnecessary, but it might have been a bug if for example an argument had been added to `cleanupHandler` that resulted in a partial application. Especially in a call that is pure side-effect. To prevent bugs like that I always use `let _: callback = ...` or `|> (ignore: callback => unit)` instead of naked `ignore`s or `let _ = ...`.

And yes, this has actually bitten me before. It's not just theoretical :grin: